protocol-specs/generators/block_processing_votes/README.md

```diff
@@ -0,0 +1,61 @@
+# Block Processing Test Generator
+
+A set of fixture generators for Transfer transactions
+
+These generators make use of an experimental library for combining transactions, include them into blocks and generate chain/account states.
+The library can be found in the file utils/chain_state_builder.js with this library it's possible to build scenarios fairly quickly with a fluent interface.
+
+Transfer Lisk:
+
+```javascript
+chainStateBuilder
+	.transfer('50')
+	.from('16313739661670634666L')
+	.to('10881167371402274308L')
+	.forge();
+```
+
+Register Delegate:
+
+```javascript
+chainStateBuilder
+	.registerDelegate('ADelegateName')
+	.for('2222471382442610527L')
+	.forge();
+```
+
+Forge invalid input block (for cases where the final block should not be included):
+
+```javascript
+chainStateBuilder
+	.transfer('0.5')
+	.from('2222471382442610527L')
+	.to('10881167371402274308L')
+	.transfer('0.5')
+	.from('2222471382442610527L')
+	.to('11325618463998518034L')
+	.forgeInvalidInputBlock();
+```
+
+Calls can also be chained:
+
+```javascript
+chainStateBuilder
```

Comment:
Yeah it could be an addition if we decide to allot time for this library (which I think we should) for now this can be accomplished with a loop but I totally get your point.